assets/src/dashboard/components/cardGrid/index.js

```diff
@@ -65,7 +65,9 @@ const CardGrid = forwardRef(function CardGrid(
       tabIndex={0}
       aria-label={ariaLabel}
       columnWidth={pageSize.width}
-      columnHeight={pageSize.containerHeight}
+      columnHeight={
```

Comment:
kinda gross but this lets the height get set based on what the ratio should be. Story poster images are a 3 : 4 ratio and stories are a 2 : 3.